CMake Visual studio build failed

I use CMake build system for build desktop project. This is the easiest way to include third-party libraries in a multi-platform project.  After this commit be43c4210f1dc648a8e8925398fda4cf1b1245ba build audio engine on win32 fails with CMake. Compile error: `cocos2d\cocos\audio/win32/AudioCache.h(35): fatal error C1083: cannot open include file: AL/al.h: No such file or directory`. Seems it wrong include of AL/al.h header file.

- cocos2d-x version: 3.14.1
- devices test on: Windows 10
- developing environments
   - VS version: 2015
   - CMake version: 3.6.2

Steps to Reproduce:

1. create blank cpp project
2. generate visual studio project by `cmake -G 'Visual Studio 14'`
3. try build generated project

--- a/cocos/audio/win32/AudioCache.h
+++ b/cocos/audio/win32/AudioCache.h
@@ -32,7 +32,11 @@
 #include <mutex>
 #include <vector>
 #include <memory>
+#ifdef OPENAL_PLAIN_INCLUDES
+#include <al.h>
+#else
 #include <AL/al.h>
+#endif
 #include "platform/CCPlatformMacros.h"
 #include "audio/apple/AudioMacros.h"
 

